sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md

```diff
@@ -0,0 +1,73 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-spotenantrenamesiteprioritization
+applicable: SharePoint Online
+title: Set-SPOTenantRenameSitePrioritization
+author: AnfraMsft
+ms.author: anfra
+ms.reviewer: anushmag
+manager: anushmag
+schema: 2.0.0
+ms.topic: reference
+---
+
+# Set-SPOTenantRenameSitePrioritization
+
+## SYNOPSIS
+This cmdlet allows prioritization of a site for early execution, as part of [Advanced Tenant Rename](https://aka.ms/advancedtenantrename).
+
+## SYNTAX
+
+```
+Set-SPOTenantRenameSitePrioritization -SiteUrl <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+As part of [Advanced Tenant Rename](https://aka.ms/advancedtenantrename), organizations can prioritize up to 4,000 sites in their scheduled rename.
```

Comment:
First add a statement that explains what the cmdlet does. Further descriptions will come after that statement.